bins/revme/src/cmd/statetest/runner.rs

```diff
@@ -372,10 +372,17 @@ pub fn execute_test_suite(
                     .and_then(Option::as_deref)
                     .cloned()
                     .unwrap_or_default();
-                let Ok(auth_list) = test.eip7702_authorization_list() else {
+
+                if unit.transaction.authorization_list.is_empty() {
```

Comment:
This is not good, it would skip all tests that do not have authorization_list

Reply:
So the current continue statement is not skipping the test which do not have authorization_list? https://github.com/bluealloy/revm/blob/9ded6908d7e81823f309251b7eb46513dbbe17b7/bins/revme/src/cmd/statetest/runner.rs#L376

Reply:
There are tests that does not have auth list, those should not be skipped but `env.tx.authorization_list` should be set to None

Reply:
thanks, applied changes!